src/client/gui/lib/tray_menu.dart

```diff
@@ -55,9 +62,7 @@ Future<void> setupTrayMenu(ProviderContainer providerContainer) async {
     await TrayMenu.instance.addLabel(
       'toggle-window',
       label: 'Toggle window',
-      callback: (_, __) async => await windowManager.isVisible()
-          ? windowManager.hide()
-          : windowManager.show(),
+      callback: (_, __) async => await windowManager.showAndRestore(),
```

Comment:
What is the purpose of this change? 'Toggle window' is supposed to hide the window if it's visible and show it if it's not.